cockroachdb/templates/role.yaml

```
@@ -20,4 +20,25 @@ rules:
     {{- else }}
     verbs: ["create", "get"]
     {{- end }}
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["get", "list", "watch"]
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ template "cockroachdb.fullname" . }}-node-reader
+  namespace: {{ .Release.Namespace | quote }}
```

```suggestion
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: {{ template "cockroachdb.clusterfullname" . }}-node-reader
```

- This ClusterRole is causing issues (see Issue #283). Please remove any namespace declaration and ensure its name is globally unique.

---

Thanks for letting us know, we've fixed this issue and released a new version `16.0.4`. Please feel free to upgrade and let us know, if you face any issues.